sdk/cosmos/azure-cosmos/azure/cosmos/_change_feed/change_feed_state.py

```diff
@@ -96,7 +108,9 @@ def from_json(
                 raise ValueError("Invalid base64 encoded continuation string [Missing version]")
 
             if version == ChangeFeedStateVersion.V2.value:
-                return ChangeFeedStateV2.from_continuation(container_link, container_rid, continuation_json)
+                change_feed_mode = change_feed_state_context.get("changeFeedMode")
+                return ChangeFeedStateV2.from_continuation(
```

Comment:
the continuation token will include changeFeedMode, so even if customer has passed in changeFeedMode, we probably should ignore it